spec/liquid/liquid_renderer_spec.rb

```diff
@@ -163,13 +163,13 @@
       it 'has the fields from one plugin form' do
         form = create :form_with_email_and_name
         create :plugins_fundraiser, page: page, form: form
-        expect(LiquidRenderer.new(page, layout: liquid_layout).personalization_data['outstanding_fields']).to eq ['email', 'name']
+        expect(LiquidRenderer.new(page, layout: liquid_layout).personalization_data['outstanding_fields']).to eq ['email', 'name', 'country']
       end
 
       it "checks with the member's liquid data" do
-        form = create :form_with_email_and_name
+        form = create :form_with_name_email_and_country
```

Comment:
OK, given that it took 4+ hours to try to straighten this test out yesterday, I'm going to make an executive decision that we're going to paper this bug with FactoryGirl over. I've changed the test so that it's testing the thing we expect it to test and the outcome is what we expect to see. The behavior of the code actually being tested is still as expected. FactoryGirl is doing something weird but I don't want to spend any more time debugging that. So, this is the compromise.